modules/calcite/src/test/java/org/apache/ignite/internal/processors/query/calcite/planner/HashAggregatePlannerTest.java

```diff
@@ -106,7 +201,7 @@ public void subqueryWithAggregate() throws Exception {
      */
     @Test
     public void noGroupByAggregate() throws Exception {
-        TestTable tbl = createAffinityTable().addIndex("val0_val1", 1, 2);
+        TestTable tbl = createAffinityTable();
```

Comment:
Do we need this change?

Reply:
Yes. Index count changes plan. Or we need to disable the rule.